test/ui/ResultLinkTest.ts

```diff
@@ -16,8 +16,6 @@ export function ResultLinkTest() {
     beforeEach(() => {
       fakeResult = initFakeResult();
       test = Mock.advancedResultComponentSetup<ResultLink>(ResultLink, fakeResult, undefined);
-      spyOn(test.cmp, 'openLink');
```

Comment:
Pretty sure this makes the UT fails, because it's now trying to do a full page reload.

Reply:
WTF Pourquoi j'ai enlever ça?!? Je corrige